packages/manager/modules/pci/src/components/project/storages/databases/node.class.js

```diff
@@ -0,0 +1,33 @@
+import includes from 'lodash/includes';
+
+export default class Node {
+  constructor({ createdAt, id, port, status, name, flavor, region }) {
+    Object.assign(this, {
+      createdAt,
+      id,
+      port,
+      status,
+      name,
+      flavor,
+      region,
+    });
+  }
+
+  get statusGroup() {
```

Comment:
it looks like the getStatusGroup could be refactored as the logic is duplicated, what about creating StatusGroup class for example ?

Reply:
Good idea! Thanks
This is done